### PR TITLE
[FLOC-3451] Ignore errors cleaning up.

### PR DESCRIPTION
**Pull request by tomprince**
It would be better to report and/or handle them, but at least ignoring them will allow other deletes to happen.

[Review on Reviewable](https://reviewable.io/reviews/clusterhq/build.clusterhq.com/147)

### CODE REVIEW

File: flocker_bb/builders/maint.py [hunk 1]
```diff
@@ -232,7 +232,10 @@ def _destroy_cloud_volumes(self, volumes):
         Unconditionally and irrevocably destroy all of the given cloud volumes.
         """
         for volume in volumes:
-            volume.destroy()
+            try:
+                volume.destroy()
+            except:
+                pass
```
Comment (sarum90) [inline flocker_bb/builders/maint.py]: As you mentioned, it'd be nice to at least log the exception so we at least know it happened, There's probably a quick way to at least do that?

**Comment by sarum90** [comment]
:+1: to cleaning up other volumes even if cleaning 1 of them up fails.

Any chance it'd be easy to add logging that the exception occurred? If not you can merge as is, but I feel like we should leave some sort of trace.

File: flocker_bb/builders/maint.py (hunk 1 above)
Comment (tomprince) [inline flocker_bb/builders/maint.py]: We could easily log it to the buildbot logs.
Getting it on the step log would be more involved, and I'm not really sure it is worth it, given that we are throwing this away soon, hopefully.

Comment (sarum90) [inline flocker_bb/builders/maint.py]: (same note below)

Comment (sarum90) [inline flocker_bb/builders/maint.py]: Note taken about buildbot not being around forever. Definitely not worth the effort to get them into the step logs.

**Pull request merged**